main/src/ca/uwaterloo/flix/language/phase/Weeder2.scala

```diff
@@ -609,29 +610,31 @@ object Weeder2 {
     private def visitEffectDecl(tree: Tree)(implicit sctx: SharedContext): Validation[Declaration.Effect, CompilationMessage] = {
       expect(tree, TreeKind.Decl.Effect)
       val ops = pickAll(TreeKind.Decl.Op, tree)
+      val mods = pickModifiers(tree, allowed = Set(TokenKind.KeywordPub))
+      val ann = pickAnnotations(tree)
       mapN(
         pickDocumentation(tree),
-        pickAnnotations(tree),
-        pickModifiers(tree, allowed = Set(TokenKind.KeywordPub)),
         pickNameIdent(tree),
         traverse(ops)(visitOperationDecl)
       ) {
-        (doc, ann, mods, ident, ops) => Declaration.Effect(doc, ann, mods, ident, ops, tree.loc)
+        (doc, ident, ops) =>
+          Declaration.Effect(doc, ann, mods, ident, ops, tree.loc)
       }
     }
 
     private def visitOperationDecl(tree: Tree)(implicit sctx: SharedContext): Validation[Declaration.Op, CompilationMessage] = {
       expect(tree, TreeKind.Decl.Op)
+      val mods = pickModifiers(tree, allowed = Set(TokenKind.KeywordPub))
+      val ann = pickAnnotations(tree)
```

Comment:
Third type of spelling :D

Reply:
Second? :))

Reply:
annotations, anns, ann?

Reply:
Hehe ok only saw ann in your comments my bad :)